providers/tentacle.rb

```diff
@@ -160,7 +162,7 @@
     action :run
     cwd tentacle_install_location
     code <<-EOH
-      .\\Tentacle.exe register-with --instance "#{instance}" --server "#{server}" --name "#{node.name}" --apiKey "#{api_key}" #{register_comm_config(polling, port)} --environment "#{environment}" #{option_list('role', roles)} --console
+      .\\Tentacle.exe register-with --instance "#{instance}" --server "#{server}" --name "#{node.name}" --apiKey "#{api_key}" #{register_comm_config(polling, port)} --environment "#{environment}" #{option_list('role', roles)} #{tenant.nil? ? '' : option_list('tenant', tenant)} #{tenanttag.nil? ? '' : option_list('tenanttag', tenanttag)} --console
```

Comment:
I think these should just default to an empty array or nil and the library should know how to output options with no values.